pkg/util/wildcard_test.go

```diff
@@ -51,6 +51,12 @@ func TestMatches(t *testing.T) {
 			candidate: "kube-system",
 			matches:   false,
 		},
+                {
+                        name:      "asterrisk on suffix and prefix",
+                        w:         Wildcard("*-kube-*"),
+                        candidate: "test-kube-test",
+                        matches:   true,
+                },
```

Comment:
can we also add a non-matching test?

Reply:
sure will add if there are hypens at the start with no asterisk